ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/provision/cucushift-installer-rehearse-aws-ipi-private-provision-chain.yaml

```diff
@@ -4,10 +4,17 @@ chain:
     - ref: ipi-conf
     - ref: aws-provision-vpc-shared
     - ref: aws-provision-bastionhost
-    - ref: ipi-conf-aws-privatecluster
+    - ref: ipi-conf-aws-custom-vpc
```

Comment:
In GCP, the step "gcp-provision-vpc" would write gcp network settings into file "${SHARED_DIR}/customer_vpc_subnets.yaml", which would be used in the step "ipi-conf-gcp". The questions, is it really necessary to add the step "ipi-conf-aws-custom-vpc" besides the existing step "ipi-conf-aws"?

Reply:
it does an option, I'd like to keep the step as simple as possible, `ipi-conf-aws-custom-vpc` just focus on subnets/AZs' settings.